iceoryx_posh/test/moduletests/test_roudi_service_registry.cpp

```diff
@@ -445,4 +465,109 @@ TEST_F(ServiceRegistry_test, AddingVariousServiceDescriptionAndGetServicesDoesNo
     EXPECT_THAT(service1Found && service2Found && service4Found, Eq(true));
 }
 
+template <typename T>
+T uniform(T max)
+{
+    static auto seed = std::random_device()();
+    static std::mt19937 mt(seed);
+    std::uniform_int_distribution<T> dist(0, max);
+    return dist(mt);
+}
+
+using string_t = iox::capro::IdString_t;
+
+string_t randomString(uint64_t size = string_t::capacity())
+{
+    // deliberately contains no `0` (need to exclude some char)
+    static const char chars[] = "123456789"
+                                "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+                                "abcdefghijklmnopqrstuvwxyz";
+
+    constexpr auto N = string_t::capacity();
+    size = std::min(N, size);
+
+    char a[N + 1];
+    for (uint64_t i = 0; i < size; ++i)
```

Comment:
Nitpick ;)

```suggestion
    for (uint64_t i = 0U; i < size; ++i)
```

Reply:
Yes but this really has to stop because it is clutter and serves no purpose (here). There is a time and place where unsigned and signed literals make a difference, but this is not one.